### ISSUE TITLE
[bug]  debian 9.13 build error

### ISSUE THREAD

**Issue by cheng990629** [tauri-apps/tauri#5684]
### Describe the bug

<img width="713" alt="image" src="https://user-images.githubusercontent.com/40415260/203796056-7daf2d87-da61-4232-8dec-0945c603dd97.png">

### Platform and versions

debian 9.13

**Comment by FabianLars** [comment]
Please _always_ include the _full_ output of `tauri info` as required by the template.

Also, try running `cargo update` in the src-tauri dir, we just released a new wry version which fixes a problem that very likely is related.

**Comment by goenning** [comment]
Sorry if I'm hijacking this tthread, but I believe it's the same root cause as the error I'm seeing. We had a brief chat about that on discord @FabianLars so I'll just add the details here as well.

```
  ...
  cargo:warning=`"pkg-config" "--libs" "--cflags" "webkit2gtk-4.0" "webkit2gtk-4.0 >= 2.36"` did not exit successfully: exit status: 1
  error: could not find system library 'webkit2gtk-4.0' required by the 'webkit2gtk-sys' crate

  --- stderr
  Requested 'webkit2gtk-4.0 >= 2.36' but version of WebKitGTK is 2.32.4
  You may find new versions of WebKitGTK at https://webkitgtk.org
```

<details>
  <summary>npx tauri info</summary>
 
  ```
npx tauri info

Environment
  › OS: Ubuntu 18.04 X64
  › Node.js: 16.17.0
  › npm: 8.15.0
  › pnpm: Not installed!
  › yarn: Not installed!
  › rustup: 1.25.1
  › rustc: 1.63.0
  › cargo: 1.63.0
  › Rust toolchain: stable-x86_64-unknown-linux-gnu 

Packages
  › @tauri-apps/cli [NPM]: 1.2.0 (outdated, latest: 1.2.1)
  › @tauri-apps/api [NPM]: 1.2.0
  › tauri [RUST]: 1.2.1,
  › tauri-build [RUST]: 1.2.1,
  › tao [RUST]: 0.15.6,
  › wry [RUST]: 0.22.5,

App
  › build-type: bundle
  › CSP: unset
  › distDir: ../dist
  › devPath: http://localhost:5173/

App directory structure
  ├─ .github
  ├─ node_modules
  ├─ examples
  ├─ .git
  ├─ dist
  ├─ src
  ├─ etc
  ├─ .vscode
  └─ src-tauri
  ```
</details>

**Comment by FabianLars** [comment]
alrighty, finally got around testing it myself. Fix is in #5697
